app/controllers/line_items_controller.rb

```diff
@@ -0,0 +1,14 @@
+class LineItemsController < ApplicationController
+  before_action :find_line_item, only: :destroy
```

Comment:
Since we have only one action, it's cleaner to simply move the code of `find_line_item` method to `destroy` action.

Reply:
fixed